pkgs/common-updater/generic-updater.nix

```diff
@@ -1,8 +1,9 @@
 { stdenv, writeScript, coreutils, gnugrep, gnused, common-updater-scripts, nix }:
 
-{ pname
-, version
-, attrPath ? pname
+{ name ? ""
+, pname ? ""
+, version ? ""
+, attrPath ? ""
```

Comment:
Maybe use `null` as a stronger signifier of “not provided”?

Reply:
Done.